tests/plugins/emoji/manual/emojimatchlabel.html

```diff
@@ -0,0 +1,12 @@
+<textarea id="editor" cols="10" rows="10">
+	<p></p>
+</textarea>
+
+<script>
+	if ( emojiTools.notSupportedEnvironment) {
```

Comment:
There is code styling issue (missing space near closing bracket).

Reply:
I guess I should enable linters in `.html` files too.